pkg/controller/plan/migration.go

```diff
@@ -599,15 +599,28 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			r.Log.Info("BeforeTransfer hook isn't ready yet")
 			return
 		}
+		var pvcNames []string
 		if r.kubevirt.isOpenstack(vm) {
-			err = r.kubevirt.createOpenStackVolumes(vm.Ref)
+			pvcNames, err = r.kubevirt.createOpenStackVolumes(vm)
 			if err != nil {
 				err = liberr.Wrap(err)
 				return
 			}
+			err = r.kubevirt.createPodToBindPVCs(vm, pvcNames)
+			if err != nil {
+				step.AddError(err.Error())
+				err = nil
+				break
+			}
 		}
 		if r.kubevirt.useOvirtPopulator(vm) {
-			err = r.kubevirt.createVolumesForOvirt(vm.Ref)
+			pvcNames, err = r.kubevirt.createVolumesForOvirt(vm)
+			if err != nil {
+				step.AddError(err.Error())
```

Comment:
why is it different than the error handling above (after the call to createOpenStackVolumes)?

Reply:
i went with the alignment we have of errors.
IMO doing it for step is failing the step while the wrapper just logs it and keep reconciling.

Reply:
@bennyz so what's the reason for doing it different for openstack above?

Reply:
looks like a mistake on my part, not something intentional

Reply:
changing the above error handling